src/LibClient.hpp

```diff
@@ -76,7 +76,7 @@ namespace libclient {
 
             [[nodiscard]] const std::map<spy::util::UUID, spy::gadget::GadgetEnum> &getEquipmentMap() const;
 
-            [[nodiscard]] const std::vector<spy::gameplay::Operation> &getOperations() const;
+            [[nodiscard]] const std::vector<std::shared_ptr<spy::gameplay::BaseOperation>> &getOperations() const;
```

Comment:
kann das const sein? also
```c++
[[nodiscard]] const std::vector<std::shared_ptr<const spy::gameplay::BaseOperation>> &getOperations() const;
```
sodass man über diese Methode nicht die Operations verändern kann.

Reply:
for more const correctness [LibCommon #138](https://github.com/SoPra-Team-17/LibCommon/issues/138) needed